src/typedocs.ts

```diff
@@ -375,8 +376,8 @@ module Main {
         });
     }
 
-    function getSymbol(node: ts.Declaration, checker: ts.TypeChecker) {
-        const symbolName = (<ts.Declaration>node).name;
+    function getSymbol(node: ts.NamedDeclaration, checker: ts.TypeChecker) {
```

Comment:
More on this [here](https://github.com/Microsoft/TypeScript/wiki/API-Breaking-Changes#typescript-24)